pkg/terraform/provider_installer.go

```diff
@@ -38,6 +38,27 @@ func (p *ProviderInstaller) Install() (string, error) {
 	providerDir := p.getProviderDirectory()
 	providerPath := p.getBinaryPath()
 
+	_, err := os.Stat(providerDir)
+	if err != nil {
+		if os.IsNotExist(err) {
+			logrus.WithFields(logrus.Fields{
+				"path": providerDir,
+			}).Debug("Provider directory destination not found, creating ...")
+			if err := os.MkdirAll(providerDir, 0755); err != nil {
+				return "", err
+			}
+		}
+	}
+
+	isDirectoryWritable, err := os.OpenFile(path.Join(providerDir, ".is_directory_writable"), os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		logrus.WithFields(logrus.Fields{
+			"path": providerDir,
+		}).Debug("Provider directory destination is not writable")
+		return "", err
+	}
+	_ = isDirectoryWritable.Close()
```

Comment:
Here we should remove the file `.is_directory_writable`

Reply:
right